userguide/content/en/docs/adding-content/shortcodes/index.md

```diff
@@ -172,22 +175,70 @@ section. It's meant to be used in combination with the other blocks shortcodes.
 
 ### alert
 
-The **alert** shortcode creates an alert block that can be used to display
-notices or warnings.
+Use the **alert** shortcode to display notices and warnings. The shortcode
+renders a [Bootstrap alert][]. It can be used with Markdown content and contain
+other shortcodes. For example:
 
 ```go-html-template
-{{%/* alert title="Warning" color="warning" */%}}
-This is a warning.
-{{%/* /alert */%}}
+{{%/* alert title="Welcome" */%}} {{%/* param greeting */%}} {{%/* /alert */%}}
 ```
 
-Renders to:
+With the `greeting` param defined as `{{% param greeting %}}`, this renders as:
 
-{{% alert title="Warning" color="warning" %}} This is a warning. {{% /alert %}}
+{{% alert title="Welcome" %}} {{% param greeting %}} {{% /alert %}}
 
-| Parameter | Default | Description                                                   |
-| --------- | ------- | ------------------------------------------------------------- |
-| color     | primary | One of the theme colors, eg `primary`, `info`, `warning` etc. |
+Parameters:
+
+- `title` (optional) Renders as a Bootstrap alert heading, at heading level 4,
+  using the `h4` Bootstrap class over a `<div>` element. This prevents the title
+  from appearing in a page's table of contents.
+- `color` (optional) names one of the [Bootstrap alert][] variants: `primary`,
```

Comment:
`color` (optional): Use this parameter to specify one of Bootstrap's predefined [alert variants][], each of which has their own color. These include `primary`, `info`, and `warning`.

(again update link target)